.travis.yml

```diff
@@ -9,10 +9,10 @@ env:
 matrix:
   fast_finish: true
   include:
-    - env: TOXENV=lint
-      python: '2.7'
-    - env: TOXENV=py27
-      python: '2.7'
+    - python: '2.7'
+      env: TOXENV=lint
```

Comment:
Perhaps python 3.5 for lint? or soon will be with this PR: https://github.com/SUSE/DeepSea/pull/1122

Reply:
I think we should use #1122 for that change since new pylint messages came up after I changed from python2 to python3 which I have fixed in that PR. I propably did not need to change this file but I think the syntax was wrong and should be fixed (even though it works if all python versions are the same)